platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBeanTest.java

```diff
@@ -562,4 +608,57 @@ public void testGetPluginsForApplication() throws Exception {
         serviceBean.getPluginsForApplication(TEST_APP_NAME).get(0),
         is(plugin1JSON));
   }
+
+  public static class StackCaptor {
+    AtomicReference<StackTraceElement[]> stackTraceElements = new AtomicReference<>();
+
+    public Stubber doCaptureStack() {
```

Comment:
✏️ Here is an idea ... capture the starting stack when doCapture() is called such that you can remove the first matching frames from your search in your matcher later.